### ISSUE TITLE
Dialog's 'X' button tooltip is getting blocked by big mouse pointers

### ISSUE THREAD

**Issue by yakkala1** [microsoft/fluentui#20152]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^8.21.1"
- **Browser and OS versions**: Windows 10 21H1; Chrome Version 94.0.4606.71 (Official Build) (64-bit)

### Describe the issue:
Dialog's Close button's hover is getting covered by large mouse pointer(size 7).
![WhatsApp Image 2021-10-07 at 11 34 50 PM](https://user-images.githubusercontent.com/32043510/136439294-0e2eedaf-1f0e-4221-9afe-8afc2713d541.jpeg)

### Please provide a reproduction of the issue in a codepen:
You can check in the [documentation](https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog).

#### Actual behavior:
Hover content is getting blocked by pointer.

#### Expected behavior:
Hover content shouldn't be getting blocked by pointer.

**Comment by gouttierre** [comment]
@yakkala1 - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows. I see the problem when the mouse size is enlarged.  

@khmakoto  - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by khmakoto** [comment]
@gouttierre @yakkala1 I spoke with @smhigley about this and the enlarged mouse pointer is not something that we should be thinking about in this context. The real problem here comes from the tooltip appearing in that button covering real space without adding any value. I'll work on a PR to remove the tooltip.
